### ISSUE TITLE
About "Inference time loss functions."

### ISSUE THREAD

**Issue by surawinfo** [OpenNMT/OpenNMT-py#639]
In the README, I noticed an  "Inference time loss functions." as a fully implemented feature. I have a couple of question: 

1) what is the purpose of the feature, is it in addition to the loss_computation for the training and validation stage  https://github.com/OpenNMT/OpenNMT-py/blob/fa0acc9413a547931ed047926f15254ac5804518/train.py#L228

2) I have tried to check in the translate.py, is the feature already implemented? 

Thanks!
